src/modules/fullscreen.js

```diff
@@ -0,0 +1,27 @@
+const requestFullscreen = (element) => {
+	if (element.requestFullscreen) {
+		element.requestFullscreen()
+	} else if (element.webkitRequestFullscreen) {
+		element.webkitRequestFullscreen()
+	} else if (element.mozRequestFullScreen) {
+		element.mozRequestFullScreen()
+	} else if (element.msRequestFullscreen) {
+		element.msRequestFullscreen()
+	}
+}
+
+const exitFullscreen = () => {
+	if (document.exitFullscreen) {
+		document.exitFullscreen()
+	} else if (document.webkitExitFullscreen) {
+		document.webkitExitFullscreen()
+	} else if (document.mozCancelFullScreen) {
+		document.mozCancelFullScreen()
+	}
+}
+
+const fullscreenElement = () => {
```

Comment:
IIRC optionally you can drop these braces and still use 2 lines for readability:

```javascript
const fullscreenElement = () =>
    document.fullscreenElement || document.webkitFullscreenElement || document.mozFullScreenElement
```

Reply:
Yeah that seems like a good change. Can we enforce this with an eslint rule?